server/services/interventionsService.test.ts

```diff
@@ -16,26 +16,27 @@ pactWith({ consumer: 'Interventions UI', provider: 'Interventions Service' }, pr
     beforeEach(async () => {
       // This is just an example for getting Pact set up; we’ll properly design this endpoint later
       await provider.addInteraction({
-        state: 'There is an existing referral with ID of 1',
+        state: 'There is an existing referral with known id',
```

Comment:
```suggestion
        state: 'There is an existing referral with ID of ac386c25-52c8-41fa-9213-fcf42e24b0b5`,
```
won't it be useful for you to know the exact ID that needs to be available when you need to set up this state in the backend Pact tests?